extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -94,6 +94,7 @@ public static function onInstantGlobalsGetVariables( array &$vars )
 	 */
 	public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$scripts". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)